installer/debian/postinst

```diff
@@ -26,6 +26,7 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		systemctl --system daemon-reload >/dev/null || true
 		if [ -n "$2" ]; then
 			_dh_action=restart
+			deb-systemd-invoke restart algorand@\* >/dev/null || true
```

Comment:
Does this work?  I don't really know offhand.

Reply:
Separately: it seems like something about `algorand@*` should go into prerm to stop all the algorand@* services

Reply:
It worked on one Ubuntu 18.04 system, I installed and upgraded and watched the algod's restart (`algorand` and `algorand@/path/to/devdir`)

Reply:
In case you missed my other comment here: what about prerm?

Reply:
added parallel clauses in prerm and postrm; I'll do some testing of uninstall today.

Reply:
confirmed `dpkg -P algorand` kills both and removes their systemd cruft